server/events/output_updater.go

```diff
@@ -128,47 +106,25 @@ func (c *ChecksOutputUpdater) UpdateOutput(ctx *command.Context, cmd PullCommand
 }
 
 func (c *ChecksOutputUpdater) handleApprovePolicies(ctx *command.Context, cmd PullCommand, res command.Result) {
-	output := c.MarkdownRenderer.Render(res, cmd.CommandName(), ctx.Pull.BaseRepo)
-	updateStatusReq := types.UpdateStatusRequest{
-		Repo:             ctx.HeadRepo,
-		Ref:              ctx.Pull.HeadCommit,
-		StatusName:       c.TitleBuilder.Build(cmd.CommandName().String()),
-		PullNum:          ctx.Pull.Num,
-		Description:      fmt.Sprintf("%s succeeded", strings.Title(cmd.CommandName().String())),
-		Output:           output,
-		State:            models.SuccessCommitStatus,
-		PullCreationTime: ctx.Pull.CreatedAt,
-	}
-
-	if err := c.VCSClient.UpdateStatus(ctx.RequestCtx, updateStatusReq); err != nil {
-		ctx.Log.Error("unable to update check run", map[string]interface{}{
-			"error": err.Error(),
-		})
-	}
-
 	// In addition, update project level atlantis/policy_check checkruns
 	for _, projectResult := range res.ProjectResults {
 		statusName := c.TitleBuilder.Build(command.PolicyCheck.String(), vcs.StatusTitleOptions{
 			ProjectName: projectResult.ProjectName,
 		})
 
-		var state models.CommitStatus
-		if projectResult.Error != nil || projectResult.Failure != "" {
-			state = models.FailedCommitStatus
-		} else {
-			state = models.SuccessCommitStatus
-		}
-
+		output := c.MarkdownRenderer.RenderProject(projectResult, command.PolicyCheck, ctx.Pull.BaseRepo)
 		updateStatusReq := types.UpdateStatusRequest{
 			Repo:             ctx.HeadRepo,
 			Ref:              ctx.Pull.HeadCommit,
 			StatusName:       statusName,
 			PullNum:          ctx.Pull.Num,
-			State:            state,
+			State:            models.SuccessCommitStatus,
```

Comment:
similar question here, remind me why we are guaranteeing success?

Reply:
Since we're not actually doing anything when approving policies, 
https://github.com/lyft/atlantis/blob/8cf99abfd260c9da2ab0db5cf9aa940e896de4be/server/events/project_command_runner.go#L203-L211
and we don't set the project level result Error or Failure, we can assume it's always going to be success.

Reply:
Added the nil check